File: app/src/main/java/com/example/esraarashad/httpurlconnectionexample/homepackage/model/AsyncResponse.java
package com.example.esraarashad.httpurlconnectionexample.homepackage.model;

import com.example.esraarashad.httpurlconnectionexample.homepackage.model.PopularPeopleModel.PeopleResults;

import java.util.ArrayList;

public interface AsyncResponse {
    void processFinish(ArrayList<PeopleResults> outputList);
}

File: app/src/main/java/com/example/esraarashad/httpurlconnectionexample/homepackage/model/AsyncResponse.kt
package com.example.esraarashad.httpurlconnectionexample.homepackage.model

import com.example.esraarashad.httpurlconnectionexample.homepackage.model.PopularPeopleModel.PeopleResults

import java.util.ArrayList

interface AsyncResponse {
    fun processFinish(outputList: ArrayList<PeopleResults>)
}
